app_legacy/Helpers/render/site-award.php

```diff
@@ -174,16 +174,21 @@ function RenderAward($award, $imageSize, $clickable = true): void
 
     if ($awardType == AwardType::Mastery) {
         if ($awardDataExtra == '1') {
-            $tooltip = "MASTERED $awardGameTitle ($awardGameConsole)";
+            $awarded = "MASTERED on $awardDate";
```

Comment:
Just a thought - should we case this normally (`Mastered`) as we did that in other places, too?

Reply:
Yea, will do.